From fbfc32bcbed59bb1130789ee7921113d7aebca8f Mon Sep 17 00:00:00 2001
From: Yanling Song <songyl@ramaxel.com>
Date: Wed, 13 Oct 2021 20:21:45 +0800
Subject: [PATCH] spraid: Add CONFIG_RAMAXEL_SPRAID in defconfig of arch arm64
 and x86
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: fbfc32bcbed59bb1130789ee7921113d7aebca8f

Ramaxel inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I494HF
CVE: NA

Add CONFIG_RAMAXEL_SPRAID in defconfig of arch arm64 and x86 to
support Ramaxel spraid controller

Signed-off-by: Yanling Song <songyl@ramaxel.com>
Reviewed-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 arch/x86/configs/openeuler_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 7325fd69e4f4..666246fa7602 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -2301,6 +2301,7 @@ CONFIG_SCSI_MPT2SAS_MAX_SGE=128
 CONFIG_SCSI_MPT3SAS_MAX_SGE=128
 CONFIG_SCSI_MPT2SAS=m
 CONFIG_SCSI_SMARTPQI=m
+CONFIG_RAMAXEL_SPRAID=m
 # CONFIG_SCSI_UFSHCD is not set
 # CONFIG_SCSI_HPTIOP is not set
 # CONFIG_SCSI_MYRB is not set
diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index a25f9210c4be..70bd39befd63 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -2318,6 +2318,7 @@ CONFIG_SCSI_MPT2SAS_MAX_SGE=128
 CONFIG_SCSI_MPT3SAS_MAX_SGE=128
 CONFIG_SCSI_MPT2SAS=m
 CONFIG_SCSI_SMARTPQI=m
+CONFIG_RAMAXEL_SPRAID=m
 # CONFIG_SCSI_UFSHCD is not set
 # CONFIG_SCSI_HPTIOP is not set
 # CONFIG_SCSI_BUSLOGIC is not set
-- 
2.26.2

